eng/pipelines/libraries/run-test-job.yml

```diff
@@ -148,6 +148,7 @@ jobs:
             # is run in a regular CI job, so there is no need to duplicate it here. So, add 'no_tiered_compilation'
             # to the 'jitstress' job instead of adding a new job just for 'no_tiered_compilation'.
 
+            # I assume we need to do something here?
```

Comment:
under the if block for `gcstress-extra` you would have to add:

```yml
- gclargepages
```

That will be passed down to the testenvironment.proj file as a Scenario parameter and then it will be matched to the item you added there and set the `JITMinOpts=1` env var